scripts/etc/init.d/rcS

```diff
@@ -848,26 +1014,62 @@ echo $hostname > /rootfs/etc/hostname || fail
 echo "OK"
 
 echo -n "  Configuring hosts... "
-touch /rootfs/etc/hosts
-# Add localhost to hosts (if needed)
+rm /rootfs/etc/hosts
+# Add localhost to hosts
 if ! grep -q "localhost" /rootfs/etc/hosts; then
```

Comment:
If you `rm /rootfs/etc/hosts` (L1017) then this `grep` is not needed, me thinks.